nt-web-app/entity/identity.ts

```diff
@@ -0,0 +1,8 @@
+export interface UserData {
```

Comment:
Existing response type definitions for the pure twitch implementation were in the entity directory, I can find a new place to move it to, but I can't put it in the `identity.ts` since it would cause a cyclical dependency.